### PR TITLE
Create getngrams

### PR DESCRIPTION
**Pull request by tsolopchuk**
It seems that I don't have access to create files in the master folder...

### CODE REVIEW

**Comment by CCorrado** [comment]
Hey @tsolopchuk, I invited you and @maackland  -> you just have to click this link to get access.

https://github.com/CCorrado/webmining/invitations

**Comment by CCorrado** [comment]
Also I just started working on this one 👍I apparently need to get an earlier start to my saturday 🥇

**Comment by tsolopchuk** [comment]
My invitation expired :(

**Comment by CCorrado** [comment]
> My invitation expired :(

Just resent!

File: getngrams [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+tagger = load(_POS_TAGGER)
+
+def processSentence(sentence,posLex,negLex,tagger):
+    
+    dict=[]
+    terms = nltk.word_tokenize(sentence)   
+        
+    tagged_terms=tagger.tag(terms)
+    
+    for i in range(len(tagged_terms)-3):
```
Comment (CCorrado) [inline getngrams]: i dont think this for loop is actually doing what you expect it to do, it isn't iterating the value of the index, but it is looping for the number of times equal to the number of words in the sentence.

image

**Pull request closed without merging**